frontend/src/store/screening.js

```diff
@@ -0,0 +1,26 @@
+const MUTATIONS = Object.freeze({
+    SET_SUCCESS: 'SET_SUCCESS'
+  })
+export const screeningState = {screeningSuccess: false} 
+export const screeningActions = {
+    async patchScreeningStudents({commit}, data){
+        const patchScreening = {
+            method: "POST",
+            headers: {
+              'Accept': 'application/json',
+              'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(data)
+          }
```

Comment:
I saw this after #60 and #61, and how good is it that we have implemented this the same way!